Change "has been removed" to "was removed".

### Motivation

E.g. 

`C-style for statement has been removed in Swift 3`

That implies you're using Swift 3. Nobody is!

### Proposed solution

Either always use "was removed", instead, or automate the conjugation to change.

### Alternatives considered

_No response_

### Additional information

_No response_

Been a while since I updated my `swift-project` folder and kept my code up to date. Getting some errors after running the following steps in order.

1. `utils/update-checkout`
2. `utils/build-script --swift-darwin-supported-archs "$(uname -m)" --xcode --clean`

Then I just wanted to run the `tests` to make sure everything was working correctly.

`utils/run-test --lit ../llvm-project/llvm/utils/lit/lit.py \
  ../build/Ninja-RelWithDebInfoAssert/swift-macosx-$(uname -m)/test-macosx-$(uname -m)`

Got the following errors below (more than one cmake error)

```
CMake Error in unittests/SourceKit/SwiftLang/CMakeLists.txt:
  Imported target "llvm_gtest_main" includes non-existent path

    "/Users/saehejkang/Documents/swift-project/llvm-project/llvm/utils/unittest/googletest/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.

-- Generating done (0.5s)
CMake Generate step failed.  Build files cannot be regenerated correctly.
FAILED: build.ninja 
/opt/homebrew/Cellar/cmake/3.27.4/bin/cmake --regenerate-during-build -S/Users/saehejkang/Documents/swift-project/swift -B/Users/saehejkang/Documents/swift-project/build/Ninja-RelWithDebInfoAssert/swift-macosx-arm64
ninja: error: rebuilding 'build.ninja': subcommand failed
ERROR: command terminated with a non-zero exit status 1, aborting
```

Are there any steps I am missing? Maybe issue is caused by updating xcode and the sdks?

--- a/include/swift/AST/DiagnosticsParse.def
+++ b/include/swift/AST/DiagnosticsParse.def
@@ -1189,7 +1189,7 @@ ERROR(do_throws_without_catch,none,
 
 // C-Style For Stmt
 ERROR(c_style_for_stmt_removed,none,
-      "C-style for statement has been removed in Swift 3", ())
+      "C-style for statement was removed in Swift 3", ())
 
 // For-each Stmt
 ERROR(expected_foreach_in,PointsToFirstBadToken,

--- a/test/Parse/recovery.swift
+++ b/test/Parse/recovery.swift
@@ -145,17 +145,17 @@ func missingWhileInRepeat() {
 func acceptsClosure<T>(t: T) -> Bool { return true }
 
 func missingControllingExprInFor() {
-  for ; { // expected-error {{C-style for statement has been removed in Swift 3}}
+  for ; { // expected-error {{C-style for statement was removed in Swift 3}}
   }
 
-  for ; // expected-error {{C-style for statement has been removed in Swift 3}}
+  for ; // expected-error {{C-style for statement was removed in Swift 3}}
   { 
   }
 
-  for ; true { // expected-error {{C-style for statement has been removed in Swift 3}}
+  for ; true { // expected-error {{C-style for statement was removed in Swift 3}}
   }
 
-  for var i = 0; true { // expected-error {{C-style for statement has been removed in Swift 3}}
+  for var i = 0; true { // expected-error {{C-style for statement was removed in Swift 3}}
     i += 1
   }
 }

--- a/test/SourceKit/DocumentStructure/structure.swift.empty.response
+++ b/test/SourceKit/DocumentStructure/structure.swift.empty.response
@@ -1685,7 +1685,7 @@
       key.column: 1,
       key.severity: source.diagnostic.severity.error,
       key.id: "c_style_for_stmt_removed",
-      key.description: "C-style for statement has been removed in Swift 3",
+      key.description: "C-style for statement was removed in Swift 3",
       key.diagnostic_stage: source.diagnostic.stage.swift.parse,
       key.ranges: [
         {

--- a/test/SourceKit/DocumentStructure/structure.swift.foobar.response
+++ b/test/SourceKit/DocumentStructure/structure.swift.foobar.response
@@ -1687,7 +1687,7 @@
       key.filepath: "-foobar",
       key.severity: source.diagnostic.severity.error,
       key.id: "c_style_for_stmt_removed",
-      key.description: "C-style for statement has been removed in Swift 3",
+      key.description: "C-style for statement was removed in Swift 3",
       key.diagnostic_stage: source.diagnostic.stage.swift.parse,
       key.ranges: [
         {

--- a/test/SourceKit/DocumentStructure/structure.swift.response
+++ b/test/SourceKit/DocumentStructure/structure.swift.response
@@ -1687,7 +1687,7 @@
       key.filepath: main.swift,
       key.severity: source.diagnostic.severity.error,
       key.id: "c_style_for_stmt_removed",
-      key.description: "C-style for statement has been removed in Swift 3",
+      key.description: "C-style for statement was removed in Swift 3",
       key.diagnostic_stage: source.diagnostic.stage.swift.parse,
       key.ranges: [
         {

--- a/test/stmt/c_style_for.swift
+++ b/test/stmt/c_style_for.swift
@@ -1,40 +1,40 @@
 // RUN: %target-typecheck-verify-swift
 
 for var i = 0; i < 10; i++ {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 for var i = 0; i < 10; i += 1 {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 for var i = 0; i <= 10; i++ {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 for var i = 0; i <= 10; i += 1 {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 for var i = 10; i > 0; i-- {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 for var i = 10; i > 0; i -= 1 {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 for var i = 10; i >= 0; i-- {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 for var i = 10; i >= 0; i -= 1 {}
-// expected-error @-1 {{C-style for statement has been removed in Swift 3}}  {{none}}
+// expected-error @-1 {{C-style for statement was removed in Swift 3}}  {{none}}
 
 let start = Int8(4)
 let count = Int8(10)
 var other = Int8(2)
 
-for ; other<count; other+=1 { // expected-error {{C-style for statement has been removed in Swift 3}} {{none}}
+for ; other<count; other+=1 { // expected-error {{C-style for statement was removed in Swift 3}} {{none}}
 }
 
-for (var number : Int8 = start; number < count; number+=1) { // expected-error {{C-style for statement has been removed in Swift 3}} {{none}}
+for (var number : Int8 = start; number < count; number+=1) { // expected-error {{C-style for statement was removed in Swift 3}} {{none}}
   print(number)
 }
 
-for (var m : Int8 = start; m < count; m+=1) { // expected-error {{C-style for statement has been removed in Swift 3}} {{none}}
+for (var m : Int8 = start; m < count; m+=1) { // expected-error {{C-style for statement was removed in Swift 3}} {{none}}
   m += 3
 }

